### PR TITLE
Enable custom port and user for mongodb + fixed typo in readme

### PR DESCRIPTION
**Pull request by jesper-friis**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,8 +49,8 @@ For production, use a public MongoDB, and follow the same instructions above for
 Run the following commands:
 
 ```shell
-docker compose pull
-docker compose up --build
+docker-compose pull
+docker-compose up --build
```
Comment (CasperWA) [inline README.md:53]: `docker compose` is the correct way to do this: 
image

This image is taken from the official Docker documentation.

File: README.md [hunk 2]
```diff
@@ -95,5 +95,5 @@ For using it with Docker, use the `--env-file .env` argument when calling `docke
 
 ## Licensing & copyright
 
-All files in this repository are [MIT licensed](LICENSE).  
+All files in this repository are [MIT licensed](LICENSE).
```
Comment (CasperWA) [inline README.md:98]: I prefer to have a new line, but not new section here. Hence the end-of-line double spaces, which does exactly this.

**Review by CasperWA — CHANGES_REQUESTED** [review]
While the affected environment variables in the docker compose file can be set explicitly and individually when running docker compose - that's the point of them, I suppose having a single environment variable that affects both would be fine in this case.

Please revert your current changes to the README.md, but instead perhaps add information about your current addition to the `docker-compose.yml` file?

File: README.md (hunk 1 above)
Comment (jesper-friis) [reply README.md:53]: Ok. Then the corresponding documentation for oteapi services should be updated.

File: README.md (hunk 2 above)
Comment (jesper-friis) [reply README.md:98]: OK. Such blanks at end of line are stripped off by default by emacs...

File: README.md (hunk 1 above)
Comment (CasperWA) [reply README.md:53]: I referenced this comment in the OTEAPI Service issue [here](https://github.com/EMMC-ASBL/oteapi-services/issues/257#issuecomment-1545829315).

**Comment by CasperWA** [comment]
While you re-requested a review, no updates have been pushed from your side to fix the requested changes, so my current review still stands.

**Review by CasperWA — CHANGES_REQUESTED** [review]
The previous review still stands.

File: README.md (hunk 1 above)
Comment (CasperWA) [inline README.md:53]:
```suggestion
docker compose pull
docker compose up --build
```

File: README.md (hunk 2 above)
Comment (CasperWA) [inline README.md:98]:
```suggestion
All files in this repository are [MIT licensed](LICENSE).  
```

**Review by CasperWA — CHANGES_REQUESTED** [review]

**Review by CasperWA — APPROVED** [review]

**Pull request merged**